src/widgets/block/padding.rs

```diff
@@ -186,7 +186,7 @@ mod tests {
     }
 
     #[test]
-    fn can_be_const() {
+    const fn can_be_const() {
```

Comment:
Yeah this case is funny. There are other can_be_const test fn which can not be const which is even more funny (because of asserts).
It’s basically pointless to have const in tests as it doesn’t bring runtime performance but it satisfies the lint. For tests I would not care about removing const when not needed.